migration.md

```diff
@@ -0,0 +1,63 @@
+# Questions
+
+This document needs to be cleaned but I thought this might be better than an infinate issue thread 😉
+
+## General
+- I see a lot of `fs` usage in the [with-mdx-remote example](https://github.com/vercel/next.js/tree/canary/examples/with-mdx-remote)
+  should this worry me? (as far as I know this is not a browser-supported package and only works locally)
+  
+- I cannot even run the [with-mdx-remote example](https://github.com/vercel/next.js/tree/canary/examples/with-mdx-remote) example what should I do?
+  
+  My steps & issue:
+  ```
+  gh clone vercel/next.js   # uses the github cli
+  cd next.js/examples/with-mdx-remote
+  yarn  # install dependencies
+  yarn add --dev eslint eslint-config-next  # to be able to use nextjs 11
```

Comment:
Remove this step and it should work fine. Clone it, pull it down, and run it if you want to reproduce.